protocol_driver_double_barrel.cc

```diff
@@ -100,22 +97,50 @@ void ProtocolDriverDoubleBarrel::HandleConnectFailure(
   barrel_2_->HandleConnectFailure(local_connection_info);
 }
 
+std::vector<TransportStat> ProtocolDriverDoubleBarrel::GetTransportStatsClient() {
+  return {};
+}
+
+std::vector<TransportStat> ProtocolDriverDoubleBarrel::GetTransportStatsServer() {
+  return {};
+}
+
+#define APPEND_VECTOR(a, b) \
```

Comment:
lets not use C-style macros for this. Try a template function or just spell it out.
At the very least you should #undef your macro away when you are done with it.

Reply:
Done